src/hooks/useErc20.ts

```diff
@@ -15,13 +15,22 @@ import { getErc20Info } from 'services/helpers'
 
 import { web3, erc20Api } from 'apps/explorer/api'
 
+import { NATIVE_TOKEN_PER_NETWORK } from 'const'
+import { isNativeToken } from 'utils'
+
 async function _fetchErc20FromNetwork(params: {
   address: string
   networkId: number
   setError: (error: string) => void
 }): Promise<SingleErc20State> {
   const { address, networkId, setError } = params
 
+  if (isNativeToken(address)) {
+    // Overwrite native address because otherwise it won't match the case
+    // Causing the caller to never know we got the token it was looking for
+    return { ...NATIVE_TOKEN_PER_NETWORK[networkId], address } || null
```

Comment:
Why `|| null`, this returns an object? Also, should the default be ETH? 
Maybe you wanted to check if the native token was defined for this network, and if not return `null`?

Reply:
Good point! I'll default to ETH and get rid of the `null`